src/commands/coverage/upload.ts

```diff
@@ -126,10 +108,6 @@ export class UploadCodeCoverageReportCommand extends Command {
   private skipGitMetadataUpload = Option.Boolean('--skip-git-metadata-upload', false)
   private gitRepositoryURL = Option.String('--git-repository-url')
 
-  private automaticReportsDiscovery = Option.String('--auto-discovery', 'true', {
-    validator: t.isBoolean(),
-    tolerateBoolean: true,
-  })
```

Comment:
isn't this a breaking change? is this intended? The `coverage` command is no longer a beta command, is it?